apps/web/src/app/globals.css

```
@@ -1,11 +1,222 @@
+    'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  --font-sans-serif: Anton, ui-sans-serif, system-ui, sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  --font-serif: Bebas Neue, ui-serif, Georgia, Cambria, 'Times New Roman',
+    Times, serif;
+}
+
+@utility container {
+  margin-inline: auto;
+  padding-inline: 0.5rem;
+
+  @media (width >= theme(--breakpoint-sm)) {
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)